reference/matrix/batch_csr_kernels.hpp

```diff
@@ -43,12 +43,6 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace gko {
 namespace kernels {
 namespace reference {
-/**
- * @brief The Compressed sparse row matrix format namespace.
- * @ref BatchCsr
- * @ingroup batch_csr
- */
-namespace batch_csr {
```

Comment:
Don't we need these kernels to be in this namespace ?

Reply:
I'm moving towards using overloading as much as possible instead of namespaces for these small "device-side" kernels. In that regard, reference is inconsistent with common/cuda. I should probably fix that now.